components/Faq/Faq.module.css

```diff
@@ -3,8 +3,12 @@
   font-family: "Nunito";
 }
 
-.faqLayout h1 {
+.faqHeader {
   margin: 1.5rem 0rem 1.5rem 2rem;
+  display: flex;
+}
+
+.faqHeader > h1 {
   font-size: 3rem;
 }
 
```

Comment:
```suggestion
.faqHeader {
  margin: 1.5rem 0rem 1.5rem 2rem;
}

.faqHeader h1 {
  font-size: 3rem;
}

```

Reply:
the > selector is more specific and thus prevents potential scoping issues which may cause css styling to overlap
I'm keeping my original